web/src/components/Dialog.tsx

```diff
@@ -26,16 +27,28 @@ export default function AlertDialog(props: IProps) {
     <div>
       <Dialog open={props.dialogIsOpen}>
         <DialogTitle>{props.title}</DialogTitle>
-        <DialogContent>
-          <DialogContentText>{props.editWarningField}</DialogContentText>
-        </DialogContent>
+        {props.editWarningField &&
+          <DialogContent>
+            <DialogContentText>{props.editWarningField}</DialogContentText>
+          </DialogContent>
+        }
         <DialogActions>
-          <Button className='dialogButton' color='secondary' onClick={props.ignoreWarning}>
-            Continue
-          </Button>
-          <Button className='dialogButton' color='primary' onClick={handleClose}>
+          <Button
+            className='dialogButton'
+            color='primary'
+            onClick={handleClose}
+            style={{ backgroundColor: theme.palette.error.main, color: 'white' }}
+          >
             Cancel
           </Button>
+          <Button
+            className='dialogButton'
+            color='primary'
+            variant='outlined'
+            onClick={props.ignoreWarning}
+          >
+            Continue
+          </Button>
```

Comment:
Can we make these small buttons inside the dialog?

Reply:
Sorry, can you explain what do you mean by that?

Reply:
I think we could possibly use smaller buttons here since the dialog is so minute, but maybe it's not necessary.